### PR TITLE
[Rec-IM] Series Creation and Propagation 

### PR DESCRIPTION
**Pull request by amos-cha**
PR tackles Series Creation Form in both the Activity Creation Form as well as a stand-alone creation button on the Activity Page. Activity's Series currently are listed above Activity Teams 

TLDR;
- Create Series prompted after Create Activity
- Create Series option on Activity Page

PR also handles ActivityType propagation

### CODE REVIEW

**Comment by amos-cha** [comment]
Current Series Listing:
Screen Shot 2023-01-13 at 7 24 57 AM

**Comment by amos-cha** [comment]
Newest Iteration (still no "beautifying")
Screen Shot 2023-01-13 at 1 59 23 PM

File: src/views/RecIM/components/Forms/CreateSeriesForm/index.js [hunk 1]
```diff
@@ -126,6 +118,9 @@ const CreateActivityForm = ({
     setErrorStatus(getCurrentErrorStatus);
   };
 
+  const isNumeric = (value) => {
+    return /^-?\d+$/.test(value) || value.length === 0;
```
Comment (silasw2) [inline src/views/RecIM/components/Forms/CreateSeriesForm/index.js:122]: nice regex 👍

File: src/views/RecIM/views/Activity/index.js [hunk 2]
```diff
@@ -15,17 +17,24 @@ import { useUser } from 'hooks';
...
 
 const Activity = () => {
   const { activityID } = useParams();
   const { profile } = useUser();
   const [loading, setLoading] = useState(true);
   const [activity, setActivity] = useState({});
+  const [openActivityForm, setOpenActivityForm] = useState(false);
   const [openCreateTeamForm, setOpenCreateTeamForm] = useState(false);
+  const [openCreateSeriesForm, setOpenCreateSeriesForm] = useState(false);
+  const subElementStyle = {
```
Comment (silasw2) [inline src/views/RecIM/views/Activity/index.js:35]: If you are leaving this here until Cam can figure out a fix for this then that is fine.

**Review by silasw2 — APPROVED** [review]
LGTM!

File: src/views/RecIM/views/Activity/index.js (hunk 2 above)
Comment (amos-cha) [reply src/views/RecIM/views/Activity/index.js:35]: I handled this in the next PR

**Review by silasw2 — APPROVED** [review]
LGTM (approving after merge)

**Pull request merged**